trie/trie_test.go

```diff
@@ -627,11 +627,13 @@ func benchGet(b *testing.B) {
 
 func benchUpdate(b *testing.B, e binary.ByteOrder) *Trie {
 	trie := NewEmpty(NewDatabase(rawdb.NewMemoryDatabase(), nil))
-	k := make([]byte, 32)
 	b.ReportAllocs()
 	for i := 0; i < b.N; i++ {
+		k := make([]byte, 32)
```

Comment:
I think we don't have to allocate the key though, the trie needs to copy it anyway afaik.